src/app/game-state/item-repo.service.ts

```diff
@@ -562,102 +562,116 @@ export class ItemRepoService {
       value: 1,
       description: 'A basic animal hide.'
     },
-    elmLog: {
```

Comment:
Can we keep both?

Reply:
I'll drop out redwood for balsa and rubywood for lignum vitae, maybe that's best.

Reply:
Sure. Not opposed to adding more wood types if you want though.